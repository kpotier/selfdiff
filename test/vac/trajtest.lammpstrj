ITEM: TIMESTEP
0
ITEM: NUMBER OF ATOMS
3
ITEM: BOX BOUNDS pp pp pp
0   1
0   1
0   1
ITEM: ATOMS type vx vy vz
1 0 0 0
2 0.5 0.5 0.5
2 -0.5 -0.5 -0.5
1 1 1 1
2 1.5 1.5 1.5
2 -1.5 -1.5 -1.5
ITEM: TIMESTEP
0
ITEM: NUMBER OF ATOMS
3
ITEM: BOX BOUNDS pp pp pp
0   1
0   1
0   1
ITEM: ATOMS type vx vy vz 
1 0.3 0.3 0.3
2 0.7 0.7 0.7
2 -0.7 -0.7 -0.7
1 1.3 1.3 1.3
2 1.7 1.7 1.7
2 -1.7 -1.7 -1.7
ITEM: TIMESTEP
0
ITEM: NUMBER OF ATOMS
3
ITEM: BOX BOUNDS pp pp pp
0   1
0   1
0   1
ITEM: ATOMS type vx vy vz
1 0.1 0.2 0.6
2 0.4 0.9 1.1
2 0.3 0.2 0.1
1 1.1 1.1 1.1
2 1.5 1.5 1.5
2 -1.5 -1.5 -1.5
