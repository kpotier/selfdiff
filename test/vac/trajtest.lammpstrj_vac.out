Integral 1.9906172839506175
2 1.1214814814814815 0.5491358024691358
4 0.8691358024691359 0.5491358024691358
